static/js/show_copied_confirmation.js

```diff
@@ -0,0 +1,20 @@
+import $ from "jquery";
+
+import {$t} from "./i18n";
+
+export function show_copied_confirmation(copy_button, clipboard) {
+    const alert_msg = $(copy_button).closest(".message_row").find(".alert-msg");
+    clipboard.on("success", () => {
+        alert_msg.text($t({defaultMessage: "Copied!"}));
+        alert_msg.css("display", "block");
+        alert_msg.addClass("copied");
+        alert_msg.delay(1000).fadeOut(300);
+        alert_msg.delay(1300).queue(function (next) {
```

Comment:
Instead of doing this, we can remove the `copied` class by passing a callback argument to `fadeOut`.